typescript/packages/subsurface-viewer/src/layers/map/mapLayer.stories.tsx

```diff
@@ -830,7 +830,7 @@ const ContourLinesComponent: React.FC<ContourLinesComponentProps> = (
 export const ContourLines: StoryObj<typeof ContourLinesComponent> = {
     args: {
         syncViewports: true,
-        show3d: true,
+        show3d: false,
```

Comment:
Why changing this ?

Reply:
I believe it originally was 2D and I don't see a point in having it 3D but I can change it if you will.

Reply:
No, it's fine.